opa/plan.sh

```diff
@@ -0,0 +1,47 @@
+gopkg="github.com/open-policy-agent/opa"
+pkg_name=opa
+pkg_description="Open Policy Agent (OPA) is a lightweight general-purpose policy engine that can be co-located with your service."
+pkg_origin=core
+pkg_version="v0.10.0"
+pkg_maintainer="Chef Software Inc. <support@chef.io>"
```

Comment:
perhaps change to `pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"`?